pkg/apis/cyclone/v1alpha1/workflow_run.go

```diff
@@ -109,6 +109,12 @@ type Status struct {
 	// A human readable message indicating details about the transition.
 	// +optional
 	Message string `json:"message,omitempty"`
+
+	// StartTime is the start time of processing stage/workflowrun
+	StartTime metav1.Time `json:"startTime,omitempty"`
+
+	// EndTime is the end time of processing stage/workflowrun
+	EndTime metav1.Time `json:"endTime,omitempty"`
```

Comment:
Use `LastTransisionTime` direclty.

Reply:
ok

Reply:
have fixed, PTAL

Reply:
`CreationTimestamp` in metadata can be used as start time? At the same time, it will be better to put `LastTransisionTime` in metadata.

Reply:
start time of overall can use the `CreationTimestamp` in metadata, but how about start time of every stage?

Reply:
I see.